airbyte-webapp/src/packages/cloud/views/auth/SignupPage/SignupPage.module.scss

```diff
@@ -1,6 +1,12 @@
 @use "../../../../../scss/colors";
 @use "../../../../../scss/variables";
 
+.container {
```

Comment:
Adding styles for Signup Page Container, as it should be responsible of positioning elements on it.